lib/rules/require-render-return.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
 
     return {
       ReturnStatement(node) {
-        const ancestors = context.getAncestors(node).reverse();
+        const ancestors = (context.sourceCode || context).getAncestors(node).reverse();
```

Comment:
let's put `getAncestors` in a util as well

Reply:
Sure. Np.